.github/workflows/aws-deploy.yaml

```
@@ -100,5 +100,10 @@ jobs:
               -e SECRET_KEY=${{secrets.AWS_SECRET_ACCESS_KEY}} \
               -e TOSS_SECRET_KEY=${{secrets.TOSS_SECRET_KEY}}\
               -e JWT_SECRET_KEY=${{secrets.JWT_SECRET_KEY}}\
+              -e RABBITMQ_HOST=${{secrets.RABBITMQ_HOST}}\
+              -e RABBITMQ_PORT=${{secrets.RABBITMQ_PORT}}\
+              -e RABBITMQ_USERNAME=${{secrets.RABBITMQ_USERNAME}}\
+              -e RABBITMQ_PASSWORD=${{secrets.RABBITMQ_PASSWORD}}\
               -p 8080:8080 "${{secrets.DOCKER_HUB_REPOSITORY}}":main
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)